README.md

```diff
@@ -4,7 +4,7 @@ This repository serves as a check-in for attendees of the **Git Workshop for Mod
 
 ---
 ## Attendees
-
+Hans Miguel S. Salazar

```

Comment:
Thank you, Hans Miguel S. Salazar. For consistency with the other PRs, I have a little nitpick here.
```suggestion
* Hans Miguel S. Salazar
```